Kconfig

```diff
@@ -71,10 +71,28 @@ config CC_COVERAGE
 
 config XEN
 	bool "Select U-Boot be run as a bootloader for XEN Virtual Machine"
-	default n
 	help
 	  Enabling this option will make U-Boot be run as a bootloader
-	  for XEN Virtual Machine.
+	  for XEN [1] Virtual Machine.
+
+	  Having a unified bootloader in initial privileged domain (Domain-0)
+	  allows resolving this:
+		1. U-boot boot scripts, which are stored on the guest’s virtual disk,
+	  are guest specific, so any change in the guest’s configuration can be
+	  handled by the guest itself.
+		2. Guest OS’ kernel can be updated if OS’ needs that without any help
+	  from Domain-0.
+		3. Using the Device Tree Overlay mechanism it is possible to customize
+	  the device tree entries yet at bootloader stage inside the guest itself,
+	  so the base device tree provided by Xen can be customized.
+
+	  Xen support for U-boot was implemented by introducing a new Xen guest
```

Comment:
We are at the very first patch, thus everything titled "was implemented" doesn't sound appropriate